vector/src/main/java/im/vector/app/features/analytics/impl/DefaultVectorAnalytics.kt

```diff
@@ -41,12 +42,12 @@ private val IGNORED_OPTIONS: Options? = null
 @Singleton
 class DefaultVectorAnalytics @Inject constructor(
         postHogFactory: PostHogFactory,
-        private val sentryFactory: SentryFactory,
+        private val sentryAnalytics: SentryAnalytics,
         analyticsConfig: AnalyticsConfig,
         private val analyticsStore: AnalyticsStore,
         private val lateInitUserPropertiesFactory: LateInitUserPropertiesFactory,
         @NamedGlobalScope private val globalScope: CoroutineScope
-) : VectorAnalytics {
+) : VectorAnalytics, ErrorTracker {
```

Comment:
Since `VectorAnalytics` extends `ErrorTracker`, there is no need to mention `ErrorTracker` again here.

Reply:
[`7323424`](https://github.com/vector-im/element-android/pull/7695/commits/7323424f3c14bd49fba152bcff808f34f7b59a65)